docs/victory-voronoi-container/docs.md

```diff
@@ -0,0 +1,132 @@
+# VictoryVoronoiContainer
+
+`VictoryVoronoiContainer` calculates a voronoi diagram based on the data of every child component it
```

Comment:
This is another case where many readers are wondering "What's would a VictoryVoronoiContainer do for me?" and a demo would be 💯